nx/exercises/exercises-1-20.livemd

```diff
@@ -0,0 +1,500 @@
+
+```elixir
+# Add your solution here.
+```
+
+<details style="margin-left: 32px">
+  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
+  <div style="padding-top: 8px">
+
+  ```elixir
+  Mix.install([{:nx, "~> 0.6"}])
+  ```
+
+  </div>
+</details>
+
+<!-- livebook:{"break_markdown":true} -->
+
+**2. Create a 1-D tensor of 10 zeros. (★☆☆)**
+
+```elixir
+# Add your solution here.
+```
+
+<details style="margin-left: 32px">
+  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
+  <div style="padding-top: 8px">
+
+  ```elixir
+  Nx.broadcast(0, {10})
+  ```
+
+  </div>
+</details>
+
+<!-- livebook:{"break_markdown":true} -->
+
+**3. Find the number of elements in `tensor`. (★☆☆)**
+
+```elixir
+tensor = Nx.tensor([[1, 2, 3], [4, 5, 6]])
+# Add your solution here.
+```
+
+<details style="margin-left: 32px">
+  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
+  <div style="padding-top: 8px">
+
+  ```elixir
+  Nx.size(tensor)
+  ```
+
+  </div>
+</details>
+
+<!-- livebook:{"break_markdown":true} -->
+
+**4. Find the number of bytes of memory in `tensor`. (★☆☆)**
+
+```elixir
+tensor = Nx.tensor([[1, 2, 3], [4, 5, 6]])
+# Add your solution here.
+```
+
+<details style="margin-left: 32px">
+  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
+  <div style="padding-top: 8px">
+
+  ```elixir
+  Nx.byte_size(tensor)
+  ```
+
+  </div>
+</details>
+
+<!-- livebook:{"break_markdown":true} -->
+
+**5a. Use `Nx.sum/2` to find the sum of all elements of `tensor`. (★☆☆)**
+
+```elixir
+tensor = Nx.tensor([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+# Add your solution here.
+```
+
+<details style="margin-left: 32px">
+  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
+  <div style="padding-top: 8px">
+
+  ```elixir
+  Nx.sum(tensor)
+  ```
+
+  </div>
+</details>
+
+<!-- livebook:{"break_markdown":true} -->
+
+**5b. Read the [documentation for `Nx.sum/2`](https://hexdocs.pm/nx/Nx.html#sum/2) then provide the correct option to sum across the _rows_. (★☆☆)**
+
+```elixir
+# Add your solution here.
+```
+
+<details style="margin-left: 32px">
+  <summary style="cursor: pointer; font-weight: bold;">Example solution</summary>
+  <div style="padding-top: 8px">
+
+  ```elixir
+  Nx.sum(tensor, axes: [1])
+  ```
+
+  <div style="padding-top: 16px">
+    <div style="background-color: rgb(62 100 255); color: white; font-weight: bold; border-top-left-radius: 8px; border-top-right-radius: 8px; padding: 8px 16px;">
+      <span style="padding-right: 4px"><i class="ri-lightbulb-line"></i></span>
+      <span>Tip</span>
+    </div>
+    <div style="border-width: 1px; border-color: rgb(62 100 255); border-bottom-left-radius: 8px; border-bottom-right-radius: 8px; padding: 0px 16px;">
+      <p>In Livebook, you can also hover over a function to display its documentation.</p>
+    </div>
+  </div>
```

Comment:
I would like to go easy on the markup, because that makes contributing to Livebook harder:

```suggestion
  <div style="padding-top: 16px">
    <i>Tip:</i> You can also hover over a function inside Livebook code cells to display its documentation.
  </div>
```

Reply:
Works for me! It honestly felt like maybe a bit much, but I figured I'd start with a 1:1 translation of the tailwind and see how we felt.